src/jobs/README.md

```diff
@@ -0,0 +1,206 @@
+
+Example Report Result:
+
+```
+expiredApps: {
+    jobName: 'EXPIRING APPLICATIONS',
+    startedAt: '2023-04-30T08:00:05.857Z',
+    finishedAt: '2023-04-30T08:00:06.473Z',
+    success: true,
+    details: { ids: ['DACO-372', 'DACO-371'], count: 2, errors: [], errorCount: 0 },
+}
+```
+
+### Close Unsubmitted Renewal Applications
+
+- file: `src/jobs/closeUnsubmittedRenewalsCheck.ts`
+- Closes any renewal applications that have not been submitted for `REVIEW` before the renewal period has ended, 90 (`DAYS_POST_EXPIRY`) days after a source application's expiry date. Once this occurs, the renewal cycle for an application is finished and an applicant would have to submit an entirely new application to regain controlled access. There is no email notification for this scenario.
+- query will retrieve all applications that are:
+  - `DRAFT`, `SIGN AND SUBMIT` or `REVISIONS REQUESTED`
+  - `renewalPeriodEndDateUtc` is before today
+  - `isRenewal` is `true`
+- sets the application state to `CLOSED`
+- adds a `CLOSED` update event to the updates list
+- does not remove the `sourceAppId` that links the renewal to original application, in order to prevent the source application from becoming renewable again.
+
+Example Report Result:
+
+```
+closedApps: {
+    jobName: 'CLOSING UNSUBMITTED RENEWALS',
+    startedAt: '2023-04-30T08:00:06.474Z',
+    finishedAt: '2023-04-30T08:00:06.655Z',
+    success: true,
+    details: { ids: ['DACO-379'], count: 1, errors: [], errorCount: 0 },
+}
+```
+
+### Approved Users List
+
+- file: `src/jobs/approvedUsersEmail.ts`
+- Retrieves a list of unique applicants and collaborators from all `APPROVED` applications in the system
+- Creates a CSV file with headers `USER NAME` (`displayName`), `OPENID` (`googleEmail`), `EMAIL` (`institutionEmail`), `CHANGED` (`lastUpdatedAtUtc`), `AFFILIATION` (`primaryAffiliation`)
+- encrypts the file
+- emails the file to EGA. A new `iv` value is generated and sent for each email, and the recipient has the encryption key necessary to decrypt on their end.
+
+Example Report Result:
+
+```
+approvedUsers: {
+    jobName: 'APPROVED USERS EMAIL',
+    startedAt: '2023-04-30T08:00:06.656Z',
+    finishedAt: '2023-04-30T08:00:06.686Z',
+    success: true,
+}
+```
+
+### Environment Variables
+
+These are the expected default values for DACO requirements, however they have been made configurable for testing convenience. They can be configured in your `.env` file.
+
+```
+# EXPIRY
+DAYS_TO_EXPIRY_1=90
+DAYS_TO_EXPIRY_2=45
+DAYS_POST_EXPIRY=90
+EXPIRY_UNIT_COUNT=years
+EXPIRY_UNIT_OF_TIME=2
+
+# ATTESTATION
+ATTESTATION_UNIT_COUNT=1
+ATTESTATION_UNIT_OF_TIME=years
+DAYS_TO_ATTESTATION=45
```

Comment:
If you want to go all the way with these docs... consider making the env variables section into a table with columns: `variable`, `description`, `type`, `default`

This would help explain what each variable is doing, and give you the chance to list which "unit of time" strings are valid and what happens if they typo this value.

Also, if you do not provide these values will the app crash or default to the above?

Reply:
The config has these defaults, but looks like i didn't mention that in the readme - will add a note about it along with the env var details!